tardis/widgets/sdec_plot.py

```diff
@@ -876,10 +876,14 @@ def generate_plot_mpl(
         self.ax.set_xlabel(r"Wavelength $[\AA]$", fontsize=12)
         if distance:  # Set y-axis label for flux
             self.ax.set_ylabel(
-                r"$F_{\lambda}$ [erg/s/cm^{2}/$\AA$]", fontsize=12
+                r"Flux [erg $\mathrm{s^{-1}}$ $\mathrm{cm^{-2}}$ $\mathrm{\AA^{-1}}$]",
+                fontsize=12,
             )
         else:  # Set y-axis label for luminosity
-            self.ax.set_ylabel(r"$L_{\lambda}$ [erg/s/$\AA$]", fontsize=12)
+            self.ax.set_ylabel(
+                r"Luminosity [erg $\mathrm{s^{-1}}$ $\mathrm{\AA^{-1}}$]",
+                fontsize=12,
+            )
```

Comment:
Same as above - roll back to L_lambda

Reply:
Sure, No problems :)

Reply:
Luminosity
![image](https://user-images.githubusercontent.com/66117751/107249775-ac488700-6a59-11eb-95a4-78cb3148f669.png)